docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '2'
+
+services:
+  pdf:
+    build:
+      context: .
+    container_name: pdf-pipeline
+    volumes:
+      - ./src:/src
+      - ./config.ini:/src/config.ini
+    networks:
+      - hubbackend_default
+
+networks:
+  hubbackend_default:
```

Comment:
sind wir sicher, dass das Netzwerk immer so heißt? Hängt das nicht davon ab, in welchem order ich docker-compose up eingeben bzw. welchen projektnamen docker-compose verwendet?

Reply:
wird in 99% der fälle so sein aber nicht immer, dachte evtl noch ne option einzubauen, den network namen zu übergeben über ein env file oder so

Reply:
ich glaube in anbetracht der Tatsache, dass das Netzwerk mittlerweile überall anders heißt würd ichs komplett aus dem compose file rauslassen und eher in die docs schreiben, wie man ein bestehendes Netzwerk anzapfen könnte.

Reply:
@burnedikt habe es abgeändert zum neuen network namen, dieser wird sich in zukunft nicht mehr ändern, da @jakob-stoeck ein `.env` file im hub-backend project root erstellt hat, welches einen festen docker-compose prpjekt namen vergibt, unabhängig vom namen des übergeordneten ordners

Reply:
solange in der readme steht, wie man das mitm netzwerk hinbekommt - alles easy. Die meiste Zeit nutzen wir eh connection per hostname.

Reply:
@burnedikt erledigt. Nachdem ich ein wenig darüber nachgedacht habe, wurde mir klar, dass man eigentlich (fast) nie netwerke hinzufügen muss. Man kann z.B. einfach LAN IP nutzen wie `192.168.0.120`.